### ISSUE TITLE
Angular Authenticator uneffective Hide feature

### ISSUE THREAD

**Issue by ksavidetove** [aws-amplify/amplify-js#1191]
Do you want to request a feature or report a bug?
Bug Report

What is the current behavior?
The [hide] property of the Angular built-in Authenticator applies on the elements, but not on the links, that send the user to nowhere.

![singin page](https://user-images.githubusercontent.com/18190507/42511831-413c4ad0-8453-11e8-8fee-387ede19de96.png)

This is the Sign In page with 'SignUp' and 'ForgotPassword' hidden. The links are still there but they redirect to nowhere.

Expected behavior?
Hide the links too.

**Comment by powerful23** [comment]
@ksavidetove Thanks. We will investigate this issue and also review you pr.

**Comment by haverchuck** [comment]
@ksavidetove  - When you were seeing this behavior were you using the "authenticator" component, or the sign-in component in isolation?

**Comment by ksavidetove** [comment]
I was using the authenticator element. So from this parent element, the default view is the Sign In, but with a link to Sign Up displayed even if I hid it. And when you use this link, the authenticator remembers that you wanted to hide this part and presents you nothing. And you are stuck there.

**Comment by ksavidetove** [comment]
Any update on this issue?

**Comment by sairathb** [comment]
Hi, I am facing a similar issue, `<amplify-authenticator [hide]="['ForgotPassword', 'SignUp']"></amplify-authenticator>` seem to be hiding the component, but the links in the Sign In component is still visible. When I click on them it shows a blank page. Is someone working to remove the links if the components are marked to hide? I am using the version 2.1.6-unstable.1 where the hide problem was fixed. Found details from the other [bug ](https://github.com/aws-amplify/amplify-js/issues/2402)

**Comment by sairathb** [comment]
I think if we pass the hide attribute to the sign-in component, then it can be accessed in the same way as in the authenticator. I can create a pull request for this, since I have the code change, but I don't have permissions
![image](https://user-images.githubusercontent.com/36173416/50520424-dd39ed80-0a85-11e9-8463-a0f9034ce16d.png)

**Comment by AGoodnight** [comment]
I find that even if I use:
`<amplify-authenticator [hide]="['SignUp','ConfirmSignUp']"></amplify-authenticator>`
It will still take me to SignUp

**Comment by jarrettj** [comment]
Yup, same issue here. Good to know it is a bug and not me going cuckoo. :)

**Comment by vjindalaurea** [comment]
Hi , any update on it ?

**Comment by powerful23** [comment]
Closing as this should have been fixed by #3173

**Comment by ddurham2** [comment]
Maybe a n00b question, but when I use ```<amplify-authenticator  [hide]="['SignUp','ConfirmSignUp']">```
I get a compile time error:
```
Error: src/app/app.component.html:1:25 - error NG8002: Can't bind to 'hide' since it isn't a known property of 'amplify-authenticator'.
1. If 'amplify-authenticator' is an Angular component and it has 'hide' input, then verify that it is part of this module.
2. If 'amplify-authenticator' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.

1 <amplify-authenticator  [hide]="['SignUp','ConfirmSignUp']">
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
If I need to import AuthenticatorComponent, it's not readily apparent the path I should import it from.

Thanks
